### DISCUSSION TITLE
Why CSP is blocking NextJS endpoints and home page to load

### DISCUSSION THREAD

**Discussion by indahud** [vercel/next.js#20158] [Help]
Hello I'm using NextJS with custom express server. For security I'm using helmetJS. But Content Security Policy is blocking NextJS endpoints and home page. Express endpoints are working fine. If I remove helmetJS then everything is working perfectly. 
There are warnings and issues in the console. It says 
```Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'"```
`Content Security Policy of your site blocks the use of 'eval' in JavaScript`
I really need helmetJS in my project.

**Comment by Sebastp** [comment]
Any solution?
